### ISSUE TITLE
OpenAPI 3.1 support issues

### ISSUE THREAD

**Issue by RomanHotsiy** [Redocly/redoc#1715]
We've released support for openapi 3.1 in [the RC.54](https://github.com/Redocly/redoc/blob/master/CHANGELOG.md#200-rc54-2021-06-09)

This is an umbrella issue to track the issues about it.

### Known issues

- `if`/`then`/`else` keywords from openapi 3.1 are not supported, https://github.com/Redocly/redoc/pull/1627
- https://github.com/Redocly/redoc/issues/1643
- `$ref` using `$id` support
- TBD

**Comment by m-mohr** [comment]
It would be great if the README could elaborate on what "basic" OpenAPI 3.1 support means. What is it lacking in comparison to the premium variant? It would help understand the limitations better.

**Comment by adamaltman** [comment]
@m-mohr the known issues are posted above. Did you find something else? If so, please report the issue and also the link into this tracking issue. Thank you.

**Comment by m-mohr** [comment]
@adamaltman No, I have no specific issues, but the README implies that there are limitations in the Redoc community edition:

![image](https://user-images.githubusercontent.com/8262166/140053437-3b98710a-3594-43c8-9c59-4614ff8a7d2c.png)

The "(basic)" implies that there are limitations or so to Redoc CE. I'm wondering what the differences are. If there are no differences, I guess the "(basic)" should be removed? Otherwise, it would be nice to document what is missing so that people can understand what is missing and whether it's worth upgrading/switching/PRing or so.

**Comment by jehanzebqayyum** [comment]
Does `propertyName` in map works in redoc?

**Comment by crossan007** [comment]
It looks like OpenAPI 3.1 added support for `scopes` in authentication schemes other than `oauth` (https://github.com/OAI/OpenAPI-Specification/issues/1366)

Using both `0.13.10` as well as the live demo, the rendered HTML file does not seem to include scopes for API Key authentication methods.

**Comment by adamaltman** [comment]
Support for `unevaluatedProperties`. 

https://github.com/Redocly/redoc/issues/1955

**Comment by AlexVarchuk** [comment]
https://github.com/Redocly/redoc/issues/1983

**Comment by AlexVarchuk** [comment]
Add support `prefixItems` and `uniqueItems` to JSON Schema

**Comment by AlexVarchuk** [comment]
https://github.com/Redocly/redoc/issues/2142

**Comment by AlexVarchuk** [comment]
https://github.com/Redocly/redoc/issues/2364

**Comment by jeremyfiel** [comment]
https://github.com/Redocly/redocly-cli/issues/1233#issue-1852036224

**Comment by 131706dulce** [comment]
https://developer.android.com/guide/topics/manifest/uses-sdk-element?hl=es-419#ApiLevels

**Comment by jeremyfiel** [comment]
> I've just run into the `$ref`/`$id` support issue. As I couldn't find anything in the issue queue I wanted to ask, _are_ there any issues I could read up on and would there be a workaround? I tried looking for a way to set up a custom schema repo for the ReDoc instance, or a way to hook into reference resolving, but there does not seem to be anything like that.

can you give an example of your $ref issue?   what, exactly, are you referring to?
